Extensions/PlatformBehavior/platformerobjectruntimebehavior.ts

```diff
@@ -1665,9 +1667,10 @@ namespace gdjs {
       this._floorLastY = this._floorPlatform!.owner.getY();
     }
 
-    beforeUpdatingObstacles(timeDelta: float) {
-      const object = this._behavior.owner;
-      //Stick the object to the floor if its height has changed.
+    doStepPostEvents(timeDelta: float) {
```

Comment:
Doing this post event is a big problem because it means your events have no chance to position anything on top of the platformer object - precisely because it runs after the events.

Any chance the fix could be done somehow else?

Reply:
I think it had to be done after the events
- events move the platform
- the character is moved accordingly
- the platform and the character are rendered without any gap

But maybe we should keep both because the platform could be moved by another behavior and the character should not fell or be stuck in this case.